src/pages/LoginPage.vue

```diff
@@ -282,7 +282,11 @@ export default defineComponent({
         }
         if ("connectAutomatically" in evbcLocalStorage) {
           connectAutomatically.value = evbcLocalStorage.connectAutomatically;
-          if (connectAutomatically.value && window.localStorage?.getItem("lastConnectedServer") !== null) {
+          if (
+              router.currentRoute.value.query.auto_connect !== 'false' &&
+              connectAutomatically.value &&
+              window.localStorage?.getItem("lastConnectedServer") !== null
```

Comment:
```suggestion
              !!window.localStorage?.getItem("lastConnectedServer")
```
Should this be a check if null or if a value is set? Then I would check for _truthy_ value instead.